### PR TITLE
fix: (Core|Platform) change image service

### PR DESCRIPTION
**Pull request by Betrozov**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/4137
#### Please provide a brief summary of this pull request.
Image service changed
#### Please check whether the PR fulfills the following requirements

- [X] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [X] tests for the changes that have been done
- [X] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [X] update `README.md`
- [X] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [X] Documentation Examples
- [X] Stackblitz works for all examples

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "hotfix-release": "./scripts/hotfix-publish-release.sh",
     "hotfix-release-latest": "./scripts/hotfix-publish-release.sh latest",
     "release:create": "create-release",
-    "start": "ng serve",
+    "start": "node --max_old_space_size=12288 ./node_modules/@angular/cli/bin/ng serve",
```
Comment (IlyaSurmay) [inline package.json:38]: plz revert

File: apps/docs/src/app/core/component-docs/tile/examples/launch-tile-example.component.html [hunk 2]
```diff
@@ -104,7 +104,7 @@ <h1 fd-tile-title>Launch Tile (Profile) Title Text</h1>
         <h2 fd-tile-subtitle>Launch Title Subtitle</h2>
     </div>
     <div fd-tile-content>
-        <span fd-tile-profile-img [backgroundImage]="'http://lorempixel.com/60/60/nature'" aria-label="Profile Image"></span>
+        <span fd-tile-profile-img [backgroundImage]="'https://picsum.photos/id/1043/60/60'" aria-label="Profile Image"></span>
```
Comment (IlyaSurmay) [inline apps/docs/src/app/core/component-docs/tile/examples/launch-tile-example.component.html:107]: unnecessary binding

File: apps/docs/src/app/core/component-docs/tile/examples/launch-tile-example.component.html [hunk 3]
```diff
@@ -119,7 +119,7 @@ <h1 fd-tile-title>Launch Tile (Profile) Title Text</h1>
         <h2 fd-tile-subtitle>Launch Title Subtitle</h2>
     </div>
     <div fd-tile-content>
-        <span fd-tile-profile-img [backgroundImage]="'http://lorempixel.com/60/60/nature'" aria-label="Profile Image"></span>
+        <span fd-tile-profile-img [backgroundImage]="'https://picsum.photos/id/1044/60/60'" aria-label="Profile Image"></span>
```
Comment (IlyaSurmay) [inline apps/docs/src/app/core/component-docs/tile/examples/launch-tile-example.component.html:122]: same

File: apps/docs/src/app/core/component-docs/tile/examples/slide-tile-example.component.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <fd-tile type="slide">
-    <div fd-tile-background-img [backgroundImage]="'http://lorempixel.com/200/200/nature'"></div>
+    <div fd-tile-background-img [backgroundImage]="'https://picsum.photos/id/1015/200/200'"></div>
```
Comment (IlyaSurmay) [inline apps/docs/src/app/core/component-docs/tile/examples/slide-tile-example.component.html:2]: same

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — APPROVED** [review]

**Review by olshansky — APPROVED** [review]

**Review by kavya-b — APPROVED** [review]

**Review by stefanoScalzo — APPROVED** [review]
LGTM

**Pull request merged**